### DISCUSSION TITLE
Sigma Validators Issue

### DISCUSSION THREAD

**Discussion by Sai-Kumar-Gandhi** [SigmaHQ/pySigma#125] [Q&A]
Traceback (most recent call last):
  File "C:\Sigma\sigma_rule_validator.py", line 3, in <module>
    from sigma.validators import validators
ImportError: cannot import name 'validators' from 'sigma.validators' (unknown location)

Code: 

import sys
from sigma.collection import SigmaCollection
from sigma.validators import validators
from sigma.validation import SigmaValidator

def main():
    if len(sys.argv) != 2:
        print("Please provide a Sigma rule file as a command line argument.")
        sys.exit(1)
    filename = sys.argv[1]
    with open(filename, 'r') as file:
        content = file.read()
    sigma_rules = SigmaCollection.from_yaml(content)
    rule_validator = SigmaValidator(validators.values())
    issues = rule_validator.validate_rules(sigma_rules)
    for issue in issues:
        print(issue)
        
if __name__ == "__main__":
    main()

**Comment by thomaspatzke** [comment]
It appears that there is an issue with your Python environment. Did you installed the *pysigma* package? Can you ensure that the *sigma* package is available by running a minimal script that only contains `import sigma`?
